apps/meteor/.mocharc.api.js

```diff
@@ -14,5 +14,6 @@ module.exports = {
 		'tests/end-to-end/api/*.js',
 		'tests/end-to-end/api/*.ts',
 		'tests/end-to-end/apps/*.js',
+		'tests/end-to-end/api/livechat/*.js',
```

Comment:
```suggestion
		'tests/end-to-end/api/livechat/*.js',
		'tests/end-to-end/api/livechat/*.ts',
```
Since there are some typescript files as well there